examples/multiagent/README.md

```diff
@@ -0,0 +1,59 @@
+# Multi-agent examples
+
+In this folder we provide a set of multi-agent example scripts using the [VMAS](https://github.com/proroklab/VectorizedMultiAgentSimulator) simulator.
+
+<p align="center">
+<img src="../media/marl_vmas.png" width="600px">
+</p>
+
+<center><i>The MARL algorithms contained in the scripts of this folder run on three multi-robot tasks in VMAS.</i></center>
+
+For more details on the experiment setup and the environments please refer to the corresponding section of the appendix in the [TorchRL paper](https://arxiv.org/abs/2306.00577).
```

Comment:
Let's make sure links are disabled in the anonymous repo

Reply:
yes they are